test/server/cards/01_SOR/units/CartelSpacer.spec.js

```diff
@@ -0,0 +1,40 @@
+describe('', function() {
+    integration(function() {
+        describe('Cartel Spacer\'s ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['cartel-spacer', 'gamorrean-guards'],
+                    },
+                    player2: {
+                        groundArena: ['atst', 'battlefield-marine', 'partisan-insurgent'],
+                    }
+                });
+            });
+            it('should not exhaust enemy unit', function () {
```

Comment:
nit: newline before test case. I've added a linter rule that will fix this now, but I'll note them in this PR so you can fix before the PR tests do